CompanyNormalization.md

```diff
@@ -55,7 +55,7 @@ Following is an example JSON response body:
         "country": "US",
         "address": "200 N La Salle St # 1100",
         "naics_description": "Employment Placement Agencies",
-        "duns_number": "095301110",
+        "duns_number": "",
```

Comment:
I would mark this with a comment // deprecated also there is an additional duns_number on the part of the response below.